server/offenceCodeDecisions/DecisionTree.test.ts

```diff
@@ -28,8 +28,7 @@ describe('decisions', () => {
         (decision.getChildren().length !== 0 && decision.getTitle() == null) ||
         (decision.getUrl() != null &&
           decision.matching(d => d !== decision && d.getUrl() === decision.getUrl()).length !== 0) ||
-        (decision.getUrl() != null && decision.getUrl().startsWith('/')) ||
-        (decision.getPage() != null && decision.getPage().startsWith('/'))
```

Comment:
Removed the page for the time being - it doesn't look like we'll need different html pages at the moment - and if we do it should be easier to put back in